### PR TITLE
[Ready for Review] Bring Back the Book!

### LINKED ISSUE #367: Delete chapters from this repository
### Summary

The book is now hosted in a different github repository: https://github.com/alan-turing-institute/the-turing-way-book and the `chapters` folder has been transferred over.

I think we need to delete the chapter from this repository now so that it doesn't confuse new contributors.

To be clear, I'm not proposing that we remove them from the history, just that we delete the folder so that it doesn't appear going forwards.

### What needs to be done?

- [ ] Everyone on (or at least a large majority of) the core team needs to acknowledge this issue - speak now or forever be super confused about where the chapters when 😜 
- [ ] A new PR to delete the chapters.

### Who can help?

@LouiseABowler @sgibson91 @rosiehigman @pherterich @alexmorley @r-j-arnold @annakrystalli @rainsworth - please add a little emoji :+1: if you're ok with this!

---

### Updates

*No updates*

### PR DESCRIPTION
**Pull request by alexmorley**
### Summary

This PR brings back all of the content from the-turing-way-book repo into this one. We were finding it difficult to manage issues and PRs across repos, and decided the confusion was not worth it.

### List of changes proposed in this PR (pull-request)

- Just moving the book back.
- Also deletes the chapters folder - fixes #367

### What should a reviewer concentrate their feedback on?

- [ ] Check that the preview site works. Url: https://5cc6b5ba5211d60008d4e452--the-turing-way-new.netlify.com
- [ ] Check that the we are happy with everything being inside a "book" folder

### Acknowledging contributors

This PR contains commits from all of the people!

By submitting this pull request I confirm that:

- [X] all contributors to this pull request who wish to be included are named in [contributors.md](https://github.com/alan-turing-institute/the-turing-way/blob/master/contributors.md).

### CODE REVIEW

**Comment by KirstieJane** [comment]
Reviewing now @alexmorley! Going to pull down locally because there are so many files.

One first review - does it make sense to have two separate `.gitignore` files? Can they just be merged to the top level?

**Comment by alexmorley** [comment]
They can. My reasoning was that I like to keep everything at the root level as tidy as possible as otherwise it can look confusing (especially as I don't think their will be any redundancy). On the other hand I appreciate that having "hidden" .gitignore's could be be confusing in itself. My rule is that one level of sub-gitignore's is fine, but no further.

Happy to change though. Not a strongly held opinion.

**Comment by KirstieJane** [comment]
Happy to go with two gitignores for now 😄 If someone gets confused then we'll merge them together ✨

File: book/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+## The Turing Way Book
```
Comment (KirstieJane) [inline book/README.md]: This readme is great - thank you!

**Comment by KirstieJane** [comment]
I'm super happy with this! Is there anyone else who might have opinions? (I know I had a very strong one about merging this back in!)

If not then lets merge! THANK YOU! 🤗

**Comment by malvikasharan** [comment]
@all-contributors please add @alexmorley for infra, maintenance, question, review

**Pull request merged**